pkg/cleaner/jobcontroller.go

```diff
@@ -95,6 +95,7 @@ func NewCleanupJob(bd *v1alpha1.BlockDevice, volMode VolumeMode, namespace strin
 		podSpec.Volumes = []v1.Volume{volume}
 	}
 
+	podSpec.ServiceAccountName = getServiceAccount()
```

Comment:
What happens if the serviceAccount name ENV is missing will the pod creation fail or use the default service account?

Reply:
if ENV is missing, we will get a `""` for the service account. Which is equal to not setting any service account.